### ISSUE TITLE
GPS tags for images

### ISSUE THREAD

**Issue by simon987** [simon987/sist2#149]
Examples

[IMG0001.jpg](https://user-images.githubusercontent.com/7120851/109404153-62084680-7931-11eb-848e-4db163082ccf.jpg)
[IMG0001.DNG](https://github.com/simon987/sist2/files/6055963/IMG0001.DNG.gz)

**Comment by simon987** [comment]
@Seednode can you help me figure out which tags are relevant? And what type of processing would be helpful to have on the values?

The latitude for example is `48 deg 56' 35.12" N` when processed through `exiftool`. I might be able to convert it to decimal (`48.94308899999999`) and spit out an URL to google maps 

```
gpsdatestamp = 2021:01:09
gpsversionid =   2,   2,   0,   0
gpslatituderef = N
gpslatitude =      48:1      , 56585399:1000000,       0:1      
gpslongituderef = E
gpslongitude =       9:1      , 28046900:1000000,       0:1      
gpsaltituderef =   0
gpsaltitude =    3163:10     
gpstimestamp =      21:1      ,       0:1      ,   22000:1000   
gpssatellites = 09
gpsstatus = A
gpsmeasuremode = 3
gpsspeedref = K
gpsspeed =       1:100    
gpstrackref = T
gpstrack =   31076:100    
gpsimgdirectionref = T
gpsimgdirection =   16580:100    
gpsmapdatum = WGS-84
gpsprocessingmethod =  65,  83,  67,  73,  73,   0,   0,   0,  71,  80,  83
```

**Comment by dpieski** [comment]
I don't think you would have to do any conversions yourself? Google will accept DMS, DMM, or DD coordinates.

A link to that gps coord above: `https://www.google.com/maps/place/48°56.585399'N+9°28.046900'E/`
You can just put in the full decimal for the minutes in the URL and Google will resolve it instead of having to convert 56.585399 (DMM)  minutes into the 56'35.1" (DMS)
And if you have it in Min/Sec, (DMS) you can use that and Google will resolve as well `https://www.google.com/maps/place/48°56'35.1"N+9°28'02.8"E/`

**Comment by simon987** [comment]
Implemented in  5b8c13fd13792bb4914e630683d6a6748f41b1e5, will be released with 2.10.1
